python_tests/test_python_frontend.py

```diff
@@ -1468,5 +1466,49 @@ def fusion_func(fd: FusionDefinition) :
             self.assertEqual(torch.real(inputs[0]), nvf_out[0])
             self.assertEqual(torch.imag(inputs[0]), nvf_out[1])
 
+
+    def test_pad(self):
+        inputs = [
+            torch.testing.make_tensor((2, 3), dtype=torch.float32, device="cuda"),
+            2,  # used as symbolic pad width input
+        ]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs[0])
+
+            # Need to do stuff like this until we merge
+            # https://github.com/csarofeen/pytorch/pull/2449
+            one = fd.define_constant(1)
+
+            t1 = fd.ops.pad(t0, [one, one, one, one])
```

Comment:
Ugly until https://github.com/csarofeen/pytorch/pull/2449 is merged/cherry-picked.

Reply:
Just merged

Reply:
Many thanks @naoyam for merging the resize PR. The comment above though refers to @jjsjann123's python number support PR.